keyboards/keebio/sinc/rev4/halconf.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2023 Danny Nguyen <danny@keeb.io>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define HAL_USE_I2C TRUE
```

Comment:
I2C enabled, but not used?

Reply:
@nooges any update to this?

Reply:
I've got breakout pins for I2C, but it's not being actively used, so should I just set to `FALSE` and let someone else override?

Reply:
I see no harm in it being configured, from my perspective it was more about making sure you had not missed anything. Others might feel differently i guess?